src/unity/toolkits/activity_classification/sequence_iterator.cpp

```diff
@@ -30,21 +30,20 @@ static std::map<std::string,size_t> generate_column_index_map(const std::vector<
 
  * \return    The most frequent value within the given vector.
  */
+
 static double vec_majority_value(const flex_vec& input_vec) {
-    size_t counter = 0;
-    double candidate = 0.0;
+    std::map<double, int> count_map;
```

Comment:
Let's use `std::vector<int>` instead of `std::map` here. We shouldn't need a map, when the indices are themselves integers (though the fact that they come into this function as doubles is quite weird).

Reply:
Or `vector<pair<int, int>>` if that makes it easier to preserve the `max_element` logic below.